recipes/pysumma/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "3.0.3" %}
+
+package:
+  name: pysumma
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pysumma/pysumma-{{ version }}.tar.gz
+  sha256: d55e5c372bf9d9b196bfbb591f0bfeac1af51eeb1f58d5b6d95403786542b067
+  
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - xarray >=0.15.0
+    - pandas
+    - netcdf4 >=1.2.5
+    - numpy >=1.11.2
+    - dask
+    - distributed
+    - toolz
+    - pytest
+    - fiona
+    - cartopy
+    - shapely
+    - seaborn
+    - matplotlib
+    - geopandas
+    - pandas
+    - pip
+    - summa >={{ version }}
+
+test:
+  imports:
+    - pysumma
+
+about:
+  home: https://github.com/UW-Hydro/pysumma
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
add comment:

```yaml
# license file manually packaged. See https://github.com/UW-Hydro/pysumma/pull/137
license_file: LICENSE
```

I've opened a PR upstream to fix having to manually package the license file (https://github.com/UW-Hydro/pysumma/pull/137). The comment in the recipe will be a visual reminder to check the status when you update the feedstock for future releases. Once the license file is included in the source distribution, you should delete the comment and remove the manually packaged copy.

Reply:
Updated